### ISSUE TITLE
Allow color-coding incidents by status

### ISSUE THREAD

**Issue by kimballa** [burningmantech/ranger-ims-server#323]
A request from an Operator in 2019: 

Color coding of incidents in the list: more please. Like ‘Incident Statuses’, color-coding can help identify at-a-glance which incidents need focus and which are just back-burnered on-hold.

I found myself repeatedly going down the list and being frustrated about having to spend precious cognitive cycles on the same on-hold incidents just identifying them as on-hold, while trying to pick out the ‘new’ or ‘on-scene’ that are currently getting radio traffic. Colors would have to be chosen carefully, and don’t go overboard, and be mindful of color-impaired users. Consider a color-tab or color-dot column rather than highlighting the whole row - not sure which is better.

**Comment by wsanchez** [comment]
Does sorting the list by status not work?

There's also a built-in way to not display on hold incidents ("show active").

Please confirm whether those have been tried and whether they don't do what you need before we try a new thing.

**Comment by kimballa** [comment]
I don't think this is a mandatory solution for an unsolvable data filtering issue. This (and other issues added yesterday) are a batch of things that came out of processing individual Operator AARs from 2019. I cannot speak to the direct circumstances that motivated this suggestion, since it didn't originate with me, but there are times when Khaki does want us to be mindful of certain on hold incidents so it's not always an option to filter them out entirely. 

It's also the case that most operators sort by id/timestamp keeping newest at the top since that's an easier way to cognitively map the list of records to the radio traffic you're experiencing.

I think this represents a UX enhancement to help navigate the text-heavy data-dense display more easily rather than a blocker.

**Comment by corprew** [comment]
i would support some sort of UI affordance here as well, the list is getting pretty crowded during busy shifts when folks are too busy to cull incidents as they go and switching context gets expensive.

**Comment by flwyd** [comment]
A colored icon (or even a unicode symbol) somewhere in the row could help make "important status" issues stand out.

**Comment by wsanchez** [comment]
Some thoughts:

 * The trick with icons is finding icons that are appropriate for each state
 * There's an existing column for status…
   * It would probably not be hard to color code the status column
